### ISSUE TITLE
Reproduce results on Waymo dataset

### ISSUE THREAD

**Issue by maudzung** [tianweiy/CenterPoint#150]
Thanks for your work.
Do I need to change any hyper-parameters in the configs before running experiments to reproduce the results of [One-stage VoxelNet](https://github.com/tianweiy/CenterPoint/blob/400ae24bb4e8f4ccabd46757738bb0304bfa2681/configs/waymo/README.md#one-stage-voxelnet)?

I executed the command:
```bash
python -m torch.distributed.launch --nproc_per_node=4 ./tools/train.py configs/waymo/voxelnet/waymo_centerpoint_voxelnet_3x.py
```

The log loss is quite high, and the outputs are too bad.
![Screenshot from 2021-06-13 21-53-15](https://user-images.githubusercontent.com/15675550/121812612-a5918100-cc92-11eb-9d4e-45b122761db8.png)

Do you have any suggestions on it? Many thanks.

**Comment by tianweiy** [comment]
The loss sounds reasonable. You don't need to change hyperparameter. I use the exact same config

**Comment by maudzung** [comment]
Ok, thanks. Let me try once again.

**Comment by tianweiy** [comment]
I will update the repo by next weekend. It should be slightly faster and achieve better result https://waymo.com/open/challenges/entry/?challenge=DETECTION_3D&email=yintianwei@utexas.edu&timestamp=1623380141003212

**Comment by maudzung** [comment]
Hi @tianweiy 
How about the  test_cfg on your side? I think `nms_iou_threshold=0.7` is quite high, so the visualized outputs are not good as the picture (The pink color is for predictions, and the blue color is the ground truth). If I decrease the `nms_iou_threshold` and increase `score_threshold`, the results are getting slightly better.
Can you please double-check? The config is [here](https://github.com/tianweiy/CenterPoint/blob/c9822e1202ff1b197db237d284f546e8cb98d766/configs/waymo/voxelnet/waymo_centerpoint_voxelnet_3x.py#L61)

![Screenshot from 2021-06-14 06-31-19](https://user-images.githubusercontent.com/15675550/121825058-91717200-ccda-11eb-8f23-1f8b4a3994ea.png)


```python
test_cfg = dict(
    post_center_limit_range=[-80, -80, -10.0, 80, 80, 10.0],
    nms=dict(
        use_rotate_nms=True,
        use_multi_class_nms=False,
        nms_pre_max_size=4096,
        nms_post_max_size=500,
        nms_iou_threshold=0.7,
    ),
    score_threshold=0.1,
    pc_range=[-75.2, -75.2],
    out_size_factor=get_downsample_factor(model),
    voxel_size=[0.1, 0.1],
)
```

**Comment by tianweiy** [comment]
0.7 gets better map. I know the visualization doesn't look good(more overlapping)

**Comment by maudzung** [comment]
Do you mean using `nms_iou_threshold=0.7` and `score_threshold=0.1` gets better results on the Waymo server? Although the visualization of outputs is not really good?

**Comment by tianweiy** [comment]
yes

**Comment by YoushaaMurhij** [comment]
@tianweiy  Congratulations for winning the 2nd place in WAD workshop!  Do you have a plan to publish your new version in the near future?

**Comment by tianweiy** [comment]
Yes, in two weeks

**Comment by lck1201** [comment]
@YoushaaMurhij  H,Bro! how do you visualize the results? Could you please share the tool...
